packages/graphback-cli/src/helpers/watch.ts

```diff
@@ -0,0 +1,22 @@
+import * as chokidar from 'chokidar'
+import * as execa from 'execa'
+import { Stats } from 'fs';
+import { checkDirectory } from './common';
+import { createResources } from './db';
+import { generateBackend } from './generate';
+
+const modelsPath = `${process.cwd()}/model`
+
+const watcher = chokidar.watch(modelsPath, {
+  ignored: /(^|[\/\\])\../,
+  persistent: true
+});
+
+export const watch = async(): Promise<void> => {
+  checkDirectory()
+  execa('npm', ['run', 'start']).stdout.pipe(process.stdout);
```

Comment:
I think that we need to cancel return object.everytime this is the reason you got stuck initially. Lets do not make changes just to het things running but resolve problems in source